pyproject.toml

```diff
@@ -47,7 +47,8 @@ ext-modules = [
 packages = ["yyjson"]
 
 [tool.cibuildwheel]
-test-command = [ 'python -c "import yyjson; print(yyjson.__package__)"' ]
+environment = { CFLAGS="-DPYPY_VERSION" }
```

Comment:
PYPY version comes from this header and is typically used to mark code that should or should be used for PyPy. See their header file here:
https://github.com/pypy/pypy/blob/release-pypy3.11-v7.3.19/pypy/module/cpyext/include/patchlevel.h#L35